MiguelAValdesG/Hackerrank/JavaMap.java

```diff
@@ -0,0 +1,32 @@
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class JavaMap {
+
+  public static void main(String[] args) {
+    Scanner readLineTerminal = new Scanner(System.in);
+    HashMap<String, Integer> phoneBook = new HashMap<>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices